libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -340,11 +338,14 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
     /** @hidden */
     private _createContent(): void {
         const content = this.tabbedContent.toArray();
-        // reset arrays
+        // reset array
         this.tabs = [];
         if (content) {
             content.forEach((contentItem) => {
                 if (!contentItem.tabLabel) {
+                    if (this.isTabbed) {
+                        throw new Error('At least one element is already tabbed, please provide a `tabLabel`');
+                    }
```

Comment:
In case when we have 2 tabs and first one without `tabLabel` and the second one with `tabLabel` the error wont be thrown.

Reply:
We need to throw the error if we have more than one tab and some of them doesn't have `tabLabel`

Reply:
fixed to first look for tab labels throughout the `tabbedContent` array, set the `isTabbed` flag and then do this validation. Now it works as expected.